emails/feedback_survey_email.Rmd

```diff
@@ -28,7 +28,7 @@ Please visit https://rse.shef.ac.uk or subscribe to our mailing list for informa
 
 <br>
 
-##### Dr `r params$lead_instructor`
+##### Mr `r params$lead_instructor`
```

Comment:
It might be appropriate to just remove the title altogether and if anyone wants to use one in future, it can be added to the `lead_instructor` parameter.